Tests/SecureXPCTests/TestHelpers.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  TestHelpers.swift
+//  
+//
+//  Created by Alexander Momchilov on 2021-11-03.
+//
+
+import XCTest
+@testable import SecureXPC
+
+// MARK: Encoding entry point
+
+func encode<T: Encodable>(_ input: T) throws -> xpc_object_t {
```

Comment:
It used to have more purpose when it was used to wrap/unwrap the payload dicts. Otherwise it's just handy to shorted the test cases, which are already quite long.